demo/lib/demo/user.ex

```diff
@@ -81,4 +82,14 @@ defmodule Demo.User do
     |> cast(attrs, @required_fields ++ @optional_fields)
     |> validate_required(@required_fields)
   end
+
+  defp validate_posts(changeset) do
+    posts = get_field(changeset, :posts) || []
+
+    if length(posts) < 1 do
+      add_error(changeset, :posts, "must have at least one post")
+    else
+      changeset
+    end
+  end
```

Comment:
Just an idea: We could provide a generic version of this via `Backpex.Ecto.ChangesetHelpers.validate_assoc_length` or something like that (in the future).